doc/tutorials/ann/ann.txt

```diff
@@ -226,14 +228,52 @@ int main()
   // Train the model.
   model.Train(trainData, trainLabels);
 
-  // Use the Predict method to get the assignments.
-  arma::mat assignments;
-  model.Predict(trainData, assignments);
+  // Use the Predict method to get the predictions.
+  arma::mat predictionTemp;
+  model.Predict(testData, predictionTemp);
+
+  /* 
+    Since the predictionsTemp is of dimensions (3 x number_of_data_points) 
+    with continuous values, we first need to reduce it to a dimension of 
```

Comment:
> Do you think it would be worthwhile to point out what each of those three dimensions mean? i.e., the predicted output class will be the dimension of `predictionsTemp` with the greatest value. I don't know what the clearest way to put that is, though.

Sure. I've added a comment about this.

Reply:
> (Longer-term thought: it might be really nice to provide some other way of getting predictions from an FFN, so that it gives back class predictions instead of the probabilities here. That way people don't have to do the postprocessing to get the predicted class labels. That probably requires a bit of engineering though, since we would need to ensure that the structure of the network was right, etc., etc.? Anyway, that's not something to do here, I just wanted to get the thought out somewhere.)

Yes, it seems more natural for the user to receive the class label than the probabilities themselves. I'll delve into this as well. Would definitely be a useful addition.

Reply:
Yeah, I guess we could call it `Classify()`, if we wanted to do that.  We would have to be clear in the documentation that calling `Classify()` only makes sense for certain types of networks.  It might be worth more discussion; if you like, feel free to open an issue for it. :)

Reply:
Sure, a discussion about this feels worthwhile. Will open up an issue soon! :)